Marlin/Configuration.h

```diff
@@ -625,7 +625,7 @@ const bool Z_MIN_PROBE_ENDSTOP_INVERTING = false; // set to true to invert the l
 
   #define XY_TRAVEL_SPEED 8000         // X and Y axis travel speed between probes, in mm/min.
 
-  #define MIN_PROBE_PT (Z_MIN_POS-2)     //How far the extruder should move down to probe
+  #define MIN_PROBE_PT (Z_MIN_POS-2)  //How far the extruder should move down to probe
```

Comment:
I know it's a minor thing, but I just know the missing space here is going to drive me crazy.